gramps/plugins/webreport/person.py

```diff
@@ -1705,19 +1715,26 @@ def display_ind_general(self):
                         table += trow
                 if self.report.options['toggle']:
                     # Show birth and/or death date if we use the close button.
+                    # If we have associated places, show them.
+                    if p_birth:
+                        p_birth = " (" + p_birth + ")"
+                    if p_death:
+                        p_death = " (" + p_death + ")"
                     if birth_date and birth_date is not Date.EMPTY:
                         trow = Html("tr") + (
                             Html("td", self._("Birth date"),
                                  class_="ColumnAttribute", inline=True),
-                            Html("td", self.rlocale.get_date(birth_date),
+                            Html("td", self.rlocale.get_date(birth_date) +
```

Comment:
This may cause a problem with RTL languages such as Hebrew.

Reply:
Done